src/k8s/query/fetch.ts

```diff
@@ -70,12 +70,20 @@ export const K8sQueryDeleteResource = <TResource extends K8sResourceCommon>(
   requestInit: K8sResourceDeleteOptions,
 ) => {
   return k8sDeleteResource<TResource>(requestInit).finally(() => {
-    !requestInit.queryOptions?.queryParams?.dryRun &&
+    if (!requestInit.queryOptions?.queryParams?.dryRun) {
```

Comment:
how about we add some comments here? so with the change, the k8sDeleteResource just delete resources when dryRun is false, right?

Reply:
this is not for deleting resource. This just removes the query from cache so it doesn't refetch data internally.